tests/Php73/Php73Test.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Php73;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ */
+class Php73Test extends TestCase
+{
+    /**
+     * @covers Symfony\Polyfill\Php73\Php73::is_countable
+     */
+    public function testIsCountable()
+    {
+        $this->assertTrue(is_countable(array(1, 2, '3')));
+        $this->assertTrue(is_countable(new \ArrayIterator(array('foo', 'bar', 'baz'))));
+        $this->assertTrue(is_countable(new \ArrayIterator()));
+        $this->assertFalse(is_countable(new \stdClass()));
+    }
+
+    /**
+     * @covers Symfony\Polyfill\Php73\Php73::is_countable
+     * @requires PHP 7.0.0
+     */
+    public function testIsCountableForGenerator()
+    {
+        $this->assertFalse(is_countable($this->genOneToTen()));
+    }
+
+    private function genOneToTen()
+    {
+        for ($i = 1; $i <= 10; ++$i) {
+            yield $i;
+        }
+    }
```

Comment:
@keradus Is there a way to add generators tests without breaking the build?

Reply:
the `yield` code would need to be extracted to dedicated file and loaded conditionally

Reply:
@keradus Done, thanks.